src/desktop/apps/order2/server.js

```diff
@@ -6,9 +6,14 @@ import { buildServerApp } from "reaction/Artsy/Router/server"
 import styled from "styled-components"
 import { buildServerAppContext } from "desktop/lib/buildServerAppContext"
 
-const app = (module.exports = express())
```

Comment:
@erikdstock do the acceptance tests run for you locally? I think you still need the `module.exports = ` here in order to be able to import this app at the root. (I'm not super familiar with this pattern, but it seems that's the case). This might also fix your testing issues.

Reply:
they were running for me... but I'll double check this.

Reply:
Well, they were running in the vscode jest runner. logging `module.exports` at the end of the file gave me a `{default: ..., orderRoute: ...}`  To remove any ambiguity I moved it into its own routes.js file.